schemas/extractor.yml

```diff
@@ -27,6 +27,44 @@ classes:
                     to this file type.
 
 
+    Usage:
+        attributes:
```

Comment:
Add `import` keyword here, if possible figure out a way to default it to the `Extractor:id`